lib-ce/emqx_dashboard/src/emqx_dashboard_admin.erl

```diff
@@ -279,20 +284,16 @@ ensure_default_user_in_db(Username) ->
 initial_default_user_passwd_hashed() ->
     case get_default_user_passwd_hashed_from_pt() of
         Empty when ?EMPTY_KEY(Empty) ->
-            %% in case it's not set yet
-            case binenv(default_user_passwd) of
-                Empty when ?EMPTY_KEY(Empty) ->
-                    error({missing_configuration, default_user_passwd});
-                Pwd ->
-                    hash(Pwd)
-            end;
+            Pwd = binenv(default_user_passwd),
```

Comment:
This can be empty, should use “public” if do not wish to crash?

Reply:
if we keep using public as default password, maybe_warn_default_pwd will always print warning, event we remove default username.

Reply:
Then mute the warning if no default user?
Using only salt as the hash input feels risky…

Reply:
I mean then the default password is empty which is a inconsistent state comparing to other implicit states in the code.

Reply:
I have changed the default password to not be empty :)